plugins/3MFReader/ThreeMFReader.py

```diff
@@ -233,8 +233,7 @@ def _read(self, file_name: str) -> Union[SceneNode, List[SceneNode]]:
                 if mesh_data is not None:
                     extents = mesh_data.getExtents()
                     if extents is not None:
-                        # We use a different coordinate space, so flip Z and Y
-                        center_vector = Vector(extents.center.x, extents.center.z, extents.center.y)
+                        center_vector = Vector(extents.center.x, extents.center.y, extents.center.z)
```

Comment:
I think we should still swap the coordinates since the 3mf spec defines a different coord system from the one we use.

Reply:
This was not an issue